src/core/fft-common.hpp

```diff
@@ -291,7 +291,8 @@ void fft_unpack_block(double *in, double *out, int start[3], int size[3],
                       int dim[3], int element);
 
 /** Debug function to print global fft mesh.
-    Print a globally distributed mesh contained in data. Element size is element.
+    Print a globally distributed mesh contained in data. Element size is
+ element.
```

Comment:
The automatic indentation was wrong here. Needs three extra spaces and stars at the beginning of the lines.

Reply:
Do you mean the "Print a globally..." paragraph should be indented by 3 extra whitespaces, i.e. start at column 8?

Reply:
`element` should be aligned with `Print`. Also there should be `*`s in the same column as the first `*` in line 293.